ui/src/components/Collection/CollectionDetail/RemoveFromCollectionBtn/index.tsx

```diff
@@ -71,12 +72,14 @@ const RemoveFromCollectionBtn = (props: IRemoveFromCollectionBtn) => {
       )}
 
       {popup ? (
-        <Modal title="Warning" onOk={handle} onCancel={handlePopup}>
-          <p>
-            This item is excluded <b>globally</b>. Removing this exclusion will
-            apply the change to all collections
-          </p>
-        </Modal>
+        <div onClick={(e) => e.stopPropagation()}>
```

Comment:
This wrapping div is not necessary. Update handle() to call e.stopPropagation() like handlePopup() does.